### PR TITLE
add ability to selectively ignore exceptions in load_neurons

### PR DESCRIPTION
**Pull request by mgeplf**
* added -ignore-exceptions option to morph_stats, to allow for ignoring bad soma

### CODE REVIEW

File: apps/morph_stats [hunk 1]
```diff
@@ -84,32 +81,38 @@ def get_parser():
                         help=('Summary output file name, if it ends in .json, '
                               'a json file is created, if .csv, then a csv file'))
 
+    parser.add_argument('-I', '--ignored-exceptions', dest='ignored_exceptions', default='',
```
Comment (arsenius7) [inline apps/morph_stats]: How would I know what to provide here?
Maybe, use `choices` here and remove `if hasattr` check on line 104?

Comment (mgeplf) [inline apps/morph_stats]: I'll make the doc better.  I don't think choices handles python classes, does it?

Comment (arsenius7) [inline apps/morph_stats]: str -> class dict?

Comment (mgeplf) [inline apps/morph_stats]: Good call, simplified the logic too.  Thanks.

**Pull request merged**